### ISSUE TITLE
Webpack loader?

### ISSUE THREAD

**Issue by PedroIssaGit** [linthtml/linthtml#364]
https://www.npmjs.com/package/htmllinter-webpack-plugin
This webpack loader, can be adopted to work with linthtml? And finally this amazing linter will have a webpack loader?

**Comment by KamiKillertO** [comment]
It's planned yes, I just didn't have the time to work on it 😅.

**Comment by PedroIssaGit** [comment]
I understand, i'm excited waiting, i had tried to make the adaptation, but does not worked and i dont have time to work in the loader for now...

**Comment by KamiKillertO** [comment]
If you have started something can you share it? Maybe I can make it work easily 🤞

**Comment by PedroIssaGit** [comment]
> If you have started something can you share it? Maybe I can make it work easily
Of course, it's my fork  https://github.com/PedroIssaGit/linthtml-async-loader of this another repository that was developed for htmllint https://github.com/nskazki/htmllint-async-loader

**Comment by PedroIssaGit** [comment]
It was useful for u?
